cmake/cmake_uninstall.cmake.in

```diff
@@ -0,0 +1,23 @@
+
+if(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+  message(FATAL_ERROR "Cannot find install manifest: @CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+endif(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+
+file(READ "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt" files)
```

Comment:
The intention is to run this file only in the build directory but not in the installed directory. In fact, this file is not installed at all. We use this file when we want to uninstall after we manually install the project as:
```
make
sudo make install
sudo make uninstall
```